gloo/transport/ibverbs/pair.cc

```diff
@@ -238,12 +248,13 @@ void Pair::sendMemoryRegion(struct ibv_mr* src, int slot) {
   // Keep memory region around until this send operation completes.
   // They are posted in FIFO order, but may complete in arbitrary order.
   // Therefore we store them in a map keyed on the buffer slot.
-  GLOO_ENFORCE_EQ(mappedSendRegions_.count(slot), 0);
-  mappedSendRegions_[slot] = std::move(mr);
+  mappedSendRegions_[slot].emplace_back(std::move(mr));
 }
 
-const struct ibv_mr* Pair::getMemoryRegion(int slot) {
-  std::unique_lock<std::mutex> lock(m_);
+void Pair::getMemoryRegion(
```

Comment:
IMHO, If we now return nothing, do you think we still name it `getMemoryRegion`. I mean when we call getSomething, we expect something will be returned?

Reply:
After reading L360, maybe we can call it recv or fetchMemoryRegion? No strong opinion..